static/css/devhub/base.less

```diff
@@ -38,14 +35,32 @@ h1 {
 }
 
 section {
-  padding: 0 20px 60px;
+  padding: 20px 20px 60px;
 
   > p {
     font-weight: 300;
     margin-top: 10px;
   }
 }
 
+// Utilities
+
+.show-large {
```

Comment:
These are slightly confusing names as I expected them to reflect the size of the contents, not when to show them.

`show-on-desktop` or `desktop-only` might be better.